scripts/approximations/rational.py

```diff
@@ -0,0 +1,47 @@
+import numpy as np
+
+def construct_vandermonde_matrix(x_list: list, y_list: list) -> list[list]:
+    """ Constructs a Vandermonde matrix for a rational approximation.
```

Comment:
Pretty sure this is not a vandermonde matrix.

Perhaps just call it rational_poly_eval_matrix

Reply:
This should be a Vandermonde matrix where the terms follow the geometric progression based on the Remco article. I also compared to the Wikipedia definition, and it seemed to be right

Reply:
You're right! Thanks for clarifying, I will update this

Reply:
Addressed